tests/unit/crypto/test_keystore.py

```diff
@@ -116,6 +116,20 @@ def test_keystore_invalid_password(tmpdir):
         _keystore = Keystore.generate('short', keystore_dir=tmpdir)
 
 
+def test_keystore_generate_report_mnemonic_true(tmpdir):
+    _keystore, words = Keystore.generate(
+        INSECURE_DEVELOPMENT_PASSWORD,
+        keystore_dir=tmpdir,
+        interactive=False,
+        report_mnemonic=True)
+    assert len(words.split(" ")) == 24
+
+
+def test_keystore_generate_report_mnemonic_blocked_by_interactive(tmpdir):
+    with pytest.raises(ValueError):
```

Comment:
If you decide to keep this exception at all, consider making the test more strict:

```suggestion
    with pytest.raises(ValueError, match="The two values: report_mnemonic and interactive,  may not both be `True`"):
```